src/Arcus.Templates.Tests.Integration/Swagger/v1/SwaggerDocAvailabilityTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Arcus.Templates.Tests.Integration.Fixture;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Arcus.Templates.Tests.Integration.Swagger.v1
+{
+    [Collection(TestCollections.Integration)]
+    [Trait("Category", TestTraits.Integration)]
+    public class SwaggerDocAvailabilityTests
+    {
+        private readonly ITestOutputHelper _outputWriter;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SwaggerDocAvailabilityTests"/> class.
+        /// </summary>
+        public SwaggerDocAvailabilityTests(ITestOutputHelper outputWriter)
+        {
+            _outputWriter = outputWriter;
+        }
+
+        [Theory]
+        [InlineData(BuildConfiguration.Debug, HttpStatusCode.OK)]
+        [InlineData(BuildConfiguration.Release, HttpStatusCode.NotFound)]
+        public async Task GetSwaggerUI_OnlyForDebugConfiguration_ReturnsOK(BuildConfiguration buildConfiguration, HttpStatusCode expectedStatusCode)
```

Comment:
This test name is confusing given it uses multiple inputs, wouldn't it be better to rename/split?